packages/components/src/InputGroup/InputGroup.css

```diff
@@ -0,0 +1,61 @@
+.vertical > *:not(:last-child) {
+  margin-bottom: -1px !important;
+}
+
+.horizontal {
+  display: flex;
+  align-items: flex-end;
+}
+
+.horizontal > *:not(:last-child) {
+  margin-bottom: auto;
+  margin-right: -1px;
+}
+
+/* Border Radius Tweak  */
+
+.inputGroup input,
+.inputGroup textarea,
+.inputGroup select {
+  border-radius: 0;
```

Comment:
Idea: form field CSS could have a `--shared-formField--border-radius` Custom property. Our something along those lines. So that it's communicated that this thing can be modified by other components

Reply:
Do we want this to be customised?